src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/jpa/entity/ApplicationEntityReportRow.kt

```diff
@@ -57,8 +57,8 @@ interface ApplicationEntityReportRowRepository : JpaRepository<ApplicationEntity
       cast(placement_applications.data -> 'request-a-placement' -> 'decision-to-release' ->> 'decisionToReleaseDate' as date) as paroleDecisionDate,
       (
         CASE
-          WHEN placement_applications.id IS NULL THEN 'referral'
-          ELSE 'placement request'
+          WHEN apa.arrival_date IS NULL THEN 'placement request'
```

Comment:
presumably this field isn't populated when the application is only used for placement requests? should we consider introducing a better marker for this on an application in a subsequent change and use this as a fall back?